static/src/server.ts

```diff
@@ -385,6 +480,32 @@ app.get("/nodejs/query", (req: Request, res: Response) => {
     });
 });
 
+app.get("/nodejs/chart", (req: Request, res: Response) => {
+  const place = _.escape(req.query[CHART_URL_PARAMS.PLACE] as string);
+  const enclosedPlaceType = _.escape(
+    req.query[CHART_URL_PARAMS.ENCLOSED_PLACE_TYPE] as string
+  );
+  const svSpec = JSON.parse(
+    req.query[CHART_URL_PARAMS.STAT_VAR_SPEC] as string
+  );
+  // Need to convert encoded # back to #.
+  const eventTypeSpecVal = (
+    req.query[CHART_URL_PARAMS.EVENT_TYPE_SPEC] as string
+  ).replaceAll("%23", "#");
+  const eventTypeSpec = JSON.parse(eventTypeSpecVal);
+  const tileConfig = JSON.parse(
+    req.query[CHART_URL_PARAMS.TILE_CONFIG] as string
+  );
+  res.setHeader("Content-Type", "text/html");
```

Comment:
Continued: this could return `image/svg+xml`

Reply:
returning image/svg+xml seems to cause a weird error .. and i think returning html here is ok?